cnd/src/network/comit_ln.rs

```diff
@@ -340,15 +459,11 @@ impl NetworkBehaviourEventProcess<announce::behaviour::BehaviourOutEvent> for Co
                         }
                     }
                     None => {
-                        tracing::warn!(
-                            "Peer {} announced a swap ({}) we don't know about",
-                            peer,
-                            io.swap_digest
+                        tracing::debug!(
+                            "Swap has not been created yet, waiting for REST API request."
```

Comment:
Log messages should adhere to module boundaries in the same way that regular code does: Nothing in this module knows about the REST API, so IMO also log messages shouldn't.

Reply:
Ok, fixing in new branch.

Reply:
The main reason is that I want to be sure everyone understand the terminology of "created" but I guess being consistent with it will be the most helpful way to fix the meaning.